tools/devops/automation/build-pipeline.yml

```diff
@@ -83,6 +84,7 @@ parameters:
       stageName: 'mac_10_14',
       displayName: 'Mac Mojave (10.14)',
       macPool: 'Hosted Mac Internal Mojave',
```

Comment:
The `Hosted Mac Internal Mojave` pool has been retired. Instead recommend using `internal-macos-10.14` where you'll set `useImage` to true

Reply:
wil do a pr with that change, thx!